src/main/kotlin/io/github/gradlenexus/publishplugin/NexusRepositoryContainer.kt

```diff
@@ -19,9 +19,17 @@ package io.github.gradlenexus.publishplugin
 import groovy.lang.Closure
 import org.gradle.api.Action
 import org.gradle.api.NamedDomainObjectContainer
+import org.gradle.internal.Actions
```

Comment:
Does it really make sense to depend on internal class here? I would prefer {} instead

Reply:
Done :+1: